packages/frontend/routes/[...slug]/index.tsx

```diff
@@ -1,33 +1,49 @@
 import { Handlers, PageProps } from "$fresh/server.ts";
-
 import { Container, ElementalArea, Head, Layout } from "@/components";
 import { sdk } from "@/utils";
-
 import type { BaseElementInterface, PageBySlugQuery } from "@/graphql";
 
-export const handler: Handlers<PageBySlugQuery> = {
-  async GET(_, ctx) {
-    const page = await sdk().PageBySlug({ slug: ctx.params.slug });
 
-    return ctx.render(page);
-  },
+export const handler: Handlers<PageBySlugQuery> = {
+    async GET(_, ctx) {
+        const { slug } = ctx.params;
+        const getPages = await sdk().PageBySlug({ slug: slug? slug : "home" });
+        return ctx.render(getPages);
+    },
 };
 
-const Page = (
-  { data: { page, navItems, siteConfig } }: PageProps<PageBySlugQuery>,
-) => (
-  <Layout headerProps={navItems} footerLinks={navItems} siteConfig={siteConfig}>
-    <Head image={undefined} page={page} siteConfig={siteConfig} />
-    <Container>
-      <h1>{page?.title}</h1>
-      {page?.elementalArea && (
-        <ElementalArea
-          elements={page?.elementalArea?.elements
-            .nodes as BaseElementInterface[]}
-        />
-      )}
-    </Container>
-  </Layout>
-);
+const Page = ({ data: { page, navItems,  siteConfig } }: PageProps<PageBySlugQuery>) => {
+
+    return (
+        <Layout headerProps={navItems} footerLinks={navItems} siteConfig={siteConfig}>
+            <Head image={undefined} page={page} siteConfig={siteConfig} />
+            <Container>
+                {
+                    page?.title && (
+                        <> 
+                            <div dangerouslySetInnerHTML={{__html: page?.content }} ></div>
```

Comment:
I think the content is being replaces by the elementals, so not sure if this is needed